chromium_src/chrome/installer/setup/sources.gni

```diff
@@ -17,7 +17,6 @@ if (is_win) {
     brave_chromium_src_chrome_installer_setup_deps += [
       "//brave/components/brave_vpn/browser/connection/ikev2/win:ras_utils",
       "//brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper:common",
-      "//brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/common",
```

Comment:
Just found that above `"//brave/components/brave_vpn/common",` should be moved here?